setup.py

```diff
@@ -119,7 +119,7 @@ def get_env_if_set(key, default: typing.Any = ""):
 # For any pre-installed ops force disable ninja.
 if torch_available:
     from accelerator import get_accelerator
-    cmdclass['build_ext'] = get_accelerator().build_extension().with_options(use_ninja=False)
+    cmdclass['build_ext'] = get_accelerator().build_extension().with_options()
```

Comment:
Can we have ninja usage control with an environment variable? It will great for users to disable ninja if needed.

Reply:
@tjruwase I have add env vars to avoid hardcoding the value of use_ninja and nvcc_threads